pkg/util/pretty/document.go

```diff
@@ -106,6 +106,24 @@ func Concat(a, b Doc) Doc {
 	})
 }
 
+// Fill concatenates a list of docs with spaces if possible,
+// otherwise with newlines.
+// See linked paper pp 14-15.
+// func Fill(d ...Doc) Doc {
```

Comment:
Was this debug work? Probably wasn't supposed to be in this commit but whatever.

Reply:
I had tried it out but then commented it out to please the linter. I think it's a nice function but we haven't used it. I was happy while writing this to see how easy it was to reproduce the algorithms!